src/pages/App.tsx

```diff
@@ -78,6 +79,7 @@ export default function App() {
             <Popups />
             <Polling />
             <TopLevelModals />
+            <ScrollToTop />
```

Comment:
This isn't presentational, so it should be implemented as a hook, and not a component. It's also top-level, so it doesn't need to be in it's own file.

Can you implement it inline in the App component as a hook?